pyproject.toml

```diff
@@ -20,13 +85,64 @@ omit = [
     "*/package/__main__.py",
 ]
 
+
 # https://pycqa.github.io/isort/
 [tool.isort]
 profile = "black"
 multi_line_output = 3
 line_length = 120
 skip_gitignore = true
 
+
+# https://mypy.readthedocs.io/en/stable/config_file.html#using-a-pyproject-toml
+[tool.mypy]
+mypy_path = "src"
+# exclude=
+show_error_codes = true
+show_column_numbers = true
+check_untyped_defs = true
+incremental = false
+warn_return_any = true
+warn_redundant_casts = true
+warn_unused_ignores = true
+
+[[tool.mypy.overrides]]
+module = "pytest"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module = "package.*"
+disallow_untyped_calls = true
+disallow_untyped_defs = true
+disallow_incomplete_defs = true
+disallow_untyped_decorators = true
+
+
+# https://pylint.pycqa.org/en/latest/user_guide/configuration/index.html
+[tool.pylint.MASTER]
+fail-under = 10.0
+disable = [
+    "fixme",
+    "too-many-statements",
+    "too-many-branches",
+    "too-many-locals",
+    "too-many-nested-blocks",
+    "too-many-return-statements",
+    "too-few-public-methods",
```

Comment:
How about these two other `too-many` warnings?

- `too-many-arguments`
- `too-many-instance-attributes`

Reply:
Commit https://github.com/jenstroeger/python-package-template/pull/208/commits/c1bf2ddd78761b27ae2eb5f107db3a1bafeb6d7d.

Maybe we should review [all messages](https://pylint.pycqa.org/en/latest/user_guide/messages/messages_overview.html) to make sure we’ve got the `too-many-*` and `too-few-*` covered?

Reply:
Naw, the other `too-many-*` kinda make more sense… let’s leave it at what we have.